### DISCUSSION TITLE
Running Wallaby tests asynchronously with Mox

### DISCUSSION THREAD

**Discussion by jonleighton** [keathley/wallaby#591] [Show and tell]
Hi! :wave: I wrote this thing which is focussed on how to run Wallaby tests asynchronously when also using Mox: https://jonleighton.name/2021/asynchronous-browser-tests-with-phoenix/

But I was pondering whether it would be useful to mention something about this approach in Wallaby's docs? I took me a bit of effort to find this solution and so I'm thinking that drawing attention to this approach may help others. I think that most projects of a certain size are likely to have some sort of resource-sharing issue like this (not necessarily Mox) so perhaps it would be helpful to highlight that switching to synchronous tests is not the only possibility.

**Comment by mhanberg** [comment]
Thank you so much for the community resource!

I've been thinking for a while that wallaby is probably in need of some guides, similar to Phoenix and Ecto.

Until I can figure something like that out, I think adding some mention of this to the README would be great. 

If you want to take a crack at it, please feel free! Otherwise I'll get it it this next week. 

Thanks!

**Reply by jonleighton** [reply]
I had a go: https://github.com/elixir-wallaby/wallaby/pull/592
